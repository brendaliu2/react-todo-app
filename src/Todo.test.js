import React from "react";
import { render } from "@testing-library/react";

import Todo from './Todo';


it("renders without crashing", function () {
  render(<Todo
    id={1}
    title='testTitle'
    description='testDesc'
    priority={1}
  />);
});

it('displays todo contents', function () {
  const { container } = render(<Todo
    id={1}
    title='testTitle'
    description='testDesc'
    priority={1}
  />);

  expect(container).toContainHTML('testTitle');
  expect(container).toContainHTML('testDesc');
  expect(container).toContainHTML('1');
});


// it("matches snapshot with default properties", function () {
//   const { container } = render(<Todo
//     id={1}
//     title='testTitle'
//     description='testDesc'
//     priority={1}
//   />);
//   expect(container).toMatchSnapshot();
// });
