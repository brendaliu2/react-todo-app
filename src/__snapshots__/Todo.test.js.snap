// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with default properties 1`] = `
<DocumentFragment>
  <div
    class="Todo"
  >
    <div>
      <b>
        testTitle
      </b>
       
      <small>
        (priority:1)
      </small>
    </div>
    <div>
      <small>
        testDesc
      </small>
    </div>
  </div>
</DocumentFragment>
`;
